src/services/services.ts

```diff
@@ -7530,7 +7533,7 @@ namespace ts {
         function walk(node: Node) {
             switch (node.kind) {
                 case SyntaxKind.Identifier:
-                    nameTable[(<Identifier>node).text] = (<Identifier>node).text;
+                    nameTable[(<Identifier>node).text] = nameTable[(<Identifier>node).text] === undefined ? node.pos : -1;

```

Comment:
If you have multiple uses, the value in the table becomes `-1` and no filtering occurs